src/messages/embed.ts

```diff
@@ -0,0 +1,260 @@
+import type {
+	APIEmbed,
+	APIEmbedAuthor,
+	APIEmbedField,
+	APIEmbedFooter,
+	APIEmbedImage,
+	APIEmbedProvider,
+	APIEmbedThumbnail,
+	APIEmbedVideo,
+} from 'discord-api-types/v8';
+
+/**
+ * Represents an embed in a message (image/video preview, rich embed, etc.)
+ */
+export class Embed implements APIEmbed {
```

Comment:
Should we name this Embed or MessageEmbed? @discordjs/the-big-3

Reply:
If `GuildEmbed` (now removed) were still around, this would be ambiguous. But since the only embeds now are from messages... 🤷🏼 

I'm fine with either.